sprout/appliances/views.py

```diff
@@ -391,7 +391,12 @@ def my_appliances(request, show_user="my"):
         end_index -= start_index
         start_index = 0
     pages = pages[start_index:end_index]
-    groups = Group.objects.order_by("id")
+    available_groups = Group.objects.filter(
```

Comment:
i recall django can add values such as this one to a query for direct use - that might be even more efficient